src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -795,14 +129,16 @@ private Object getIdString() {
             Field idField = this.getClass().getDeclaredField("id");
             return idField.get(this);
         } catch (SecurityException e) {
-            return "";
+            e.printStackTrace();
```

Comment:
I'm not sure we want to do this. Printing the stack trace isn't as useful as raising an actual error here saying the object has no ID.

Reply:
I think the legacy code just ignores the error and returns an empty string for `id`. To stay the same logic, we could keep the function working which we just return an empty string and print out the error stack trace

Reply:
We don't use `printStackTrace` anywhere else in the lib. It's meant for debugging purposes. Since this library will be used by end-users and not us, we should either leave it how it was or instead raise an actual error here with a helpful human-readable message. Sending through the whole stack trace is too messy and is unwanted from a library.

Reply:
Code is reverted